slide-deck.js

```diff
@@ -210,17 +132,18 @@ class slideDeck extends HTMLElement {
 
     // shadow dom and ID
     slideDeck.appendShadowTemplate(this);
-    slideDeck.adoptShadowStyles(this);
     this.setDeckID();
 
     // relevant nodes
     this.body = document.querySelector('body');
-    this.controlPanel = this.shadowRoot.querySelector(`[part="controls"]`);
+    this.controlPanel = this.querySelector(`[slot="control-panel"]`) ??
+      this.shadowRoot.querySelector(`[part="control-panel"]`);
 
     // initial setup
-    this.slideCount = this.childElementCount;
+    const slides = this.querySelectorAll(':scope > :not([slot])');
```

Comment:
@jamesnw what do you think about making this a bit more specific?

```suggestion
    const slides = this.querySelectorAll(':scope > :not([slot=control-panel])');
```

Reply:
@mirisuzanne I intentionally made that more generic. I don't think we have plans for additional slots, but this future proofs it slightly. I think it also matches the browser behavior a bit closer- Slides are the direct children in the default slot, and the default slot contains everything that is not slotted.

Alternatively, could we query the shadow dom default slot content directly? Then we are directly measuring what we want, but I'm not quickly seeing how to do that.

Reply:
That makes sense, let's leave it as-is for now.